src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -609,10 +609,18 @@ function initializeEnvironmentalVariables(frb: FunctionsRuntimeBundle): void {
     // Ignore, config is optional
   }
 
+  let emulatedDatabaseURL = undefined;
+  if (frb.emulators.database) {
```

Comment:
TODO(samstern): This needs to be dependent on a specific version of the `firebase-functions` SDK.  Will have to wait until the functions PR ships to know that version.